safe_control_gym/controllers/base_controller.py

```diff
@@ -127,3 +127,67 @@ def load(self,
     def setup_results_dict(self):
         '''Setup the results dictionary to store run information. '''
         self.results_dict = {}
+        
+    def get_prior(self, env, prior_info={}):
+        '''Fetch the prior model from the env for the controller.
+
+        Note there's a default env.symbolic when each each env is created.
+        To make a different prior model, do the following when initializing a ctrl::
+
+            self.env = env_func() 
+            self.model = self.get_prior(self.env)
+
+        Besides the env config `base.yaml` and ctrl config `mpc.yaml`, 
```

Comment:
why dont we just add the prior info to the controller yaml directly? and why is the defauly controller yaml `mpc.yaml`

Reply:
i think adam mentions if multiple controllers want to use the same prior, then it's better to have a yaml for it and use it in `--overrides` so we don't need to copy and paste the prior configs to each ctrl config.   the `mpc.yaml` is an example, i intend to say if there's custom prior for a controller generally we can do `--overrides base.yaml ctrl.yaml prior.yaml` to specify the configs for env, controller and prior

Reply:
Ah ok makes sense!